index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>A Meaningful Page Title</title>
+    <link href="style.css" rel="stylesheet" />
+  </head>
+
+  <body>
+    <div id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <div id="img-div">
+        <img id="image" src="images/10613180113_fdf7bcd316_b.jpg" />
+        <p id="img-caption">
+          Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+          on how to increase wheat yields - part of his life-long war on hunger.
+        </p>
+      </div>
+      <div>
```

Comment:
- [ ] Elements wrapped inside `main` should use `section` tag, please change `div` to `section`